variables.tf

```diff
@@ -1,4 +1,170 @@
-variable "example" {
-  description = "Example variable"
-  default     = "hello world"
+#terraform {
```

Comment:
remove commented lines, or add comment for when experimental features are required (pre tf v 1.3)

Reply:
this experiment was canceled. there's now official support for optional parameters in teraform 1.3, which i see you're already using.